README.md

```diff
@@ -1,34 +1,39 @@
-# xi editor
+<h1 align="center">
+  <br>
```

Comment:
I don't understand why such a big emphasis on "big" because the size is not really that big, its literally just a <br> tag (break line). It looks abit tight with no <br> tag.